example/Src/main.c

```diff
@@ -413,6 +413,61 @@ void HAL_GPIO_EXTI_Callback(uint16_t GPIO_Pin)
     	}
     }
 }
+
+/*@brief this fuction does One-Shot for MCP3564 */
+void ADC_One_Shot(SPI_HandleTypeDef *hspi)

```

Comment:
`ADC_One_Shot` is not called anywhere, but much more importantly, it is not defined before.

Reply:
> I don't think the code like this runs or even compiles. But it has good work in it that can be adapted.

I compiles on my machine. But thais not a good reason ;)